README_zh.md

```diff
@@ -0,0 +1,57 @@
+[TOC]
+
+# 自适应限流概述
+
+## 特点
+
+1. 使用方便，只需提交`SmartLimiter`资源即可达到服务限流的目的。
+2. 自适应限流，根据服务的资源使用量等指标动态的触发限流规则。
+
+## 背景
```

Comment:
it seems that a lot of content is lost compared to https://github.com/slime-io/slime/blob/master/doc/zh/smart_limiter.md